### PR TITLE
Packaging & testing fixes

### PR DESCRIPTION
**Pull request by akx**
I attempted to use Binwalk OOTB on Windows, to not much success. This fixes some of that stuff (hopefully without breaking anything, too).

* Use standard setuptools bits (entry points, manifest.in)
* Remove some dead code
* Use py.test for testing

### CODE REVIEW

**Comment by Kungergely** [comment]
Thanks a LOT for this PR! It finally makes binwalk work properly on Windows 10 too without doing lots of hacking to make it work. Really appreciate it!

**Comment by akx** [comment]
@devttys0 Could you take a look at this? Apparently it helps others so would be nice to get merged...

**Comment by akx** [comment]
@devttys0 I've rebased this on top of `master` for your consideration. You had apparently independently done some of the work I had done in this PR in #492 and #493... :)

File: setup.py [hunk 1]
```diff
@@ -25,17 +25,11 @@
...
 
-# Python2/3 compliance
-try:
-    raw_input
-except NameError:
-    raw_input = input
-
 
 def which(command):
     # /usr/local/bin is usually the default install path, though it may not be in $PATH
```
Comment (Ayowel) [inline setup.py:35]: Might want to add a `location=None` here

**Review by Ayowel — COMMENTED** [review]
Hijacking this PR as I just saw #501 for windows

**Comment by akx** [comment]
@devttys0 Rebased once more.

**Comment by devtty1er** [comment]
@akx thanks for this, it helped me!
@devttys0 it would be great to get some feedback on this PR.

**Comment by akx** [comment]
Rebased for the new year, with additional fixes for Windows compatibility.

**Comment by akx** [comment]
Happy 2023! Rebased once again 🎉 

Could e.g. @devttys0 take a look? 😁

**Comment by akx** [comment]
Happy 2024! 🎆 Didn't rebase, because there's nothing happening in this repo!

**Comment by stkw0** [comment]
Given that the project seems to be almost abandoned and that I could not find an email address for the main author I decided to make a fork to apply maintenance patches: https://github.com/OSPG/binwalk

I merge this PR on the fork

**Comment by jowinjohnchemban** [comment]
Is the official Windows build available @devttys0

**Pull request closed without merging**